frappe/utils/boilerplate.py

```diff
@@ -634,6 +634,9 @@ def _create_parent_folder_if_not_exists(self):
           restore-keys: |
             ${{{{ runner.os }}}}-yarn-
 
+      - name: Install MariaDB Client
+        run: sudo apt-get install mariadb-client-10.6
```

Comment:
Remove mysql or it won't install due to conflicts.

Reply:
Worked for me without uninstall 🤔